# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/streams/idlharness-shadowrealm.window.js.ini
# Commit: 68ca2f48ea95
# Full Hash: 68ca2f48ea95576a330906ffa0f860c2d783f429
# Author: Tom Schuster <evilpies@gmail.com>
# Date: 2022-12-07 04:11:18
# Regressor Bug: 1734244
# File Overlap Count: 1
# Description:
#   Bug 1734244 - Implement async iteration of ReadableStream. r=mgaudet,peterv
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D153312
# ==============================================================================

diff -r 1d859fd9522d -r 68ca2f48ea95 testing/web-platform/meta/streams/idlharness-shadowrealm.window.js.ini
--- a/testing/web-platform/meta/streams/idlharness-shadowrealm.window.js.ini	Tue Dec 06 12:32:58 2022 +0000
+++ b/testing/web-platform/meta/streams/idlharness-shadowrealm.window.js.ini	Tue Dec 06 12:46:44 2022 +0000
@@ -1,6 +1,2 @@
 [idlharness-shadowrealm.window.html]
   prefs: [javascript.options.experimental.shadow_realms:true]
-
-  [ReadableStream interface: async iterable<any>]
-    bug: https://bugzilla.mozilla.org/show_bug.cgi?id=1734244
-    expected: FAIL